examples/panic/panic.description

```diff
@@ -1,8 +1,8 @@
-// In Ballerina, throwing an error means something unexpected has occured
-// (something which should not have occurred during the normal operation). A thrown error causes
+// In Ballerina, panicking an error means something unexpected has occured
```

Comment:
Shall we refrain from using terms like "panicking an error", "panicked error" and simply refer to it as a panic? e.g., https://gobyexample.com/panic